app/controllers/application_controller.rb

```diff
@@ -202,15 +200,4 @@ def signed_in?
   def page_not_found
     render file: "#{Rails.root}/public/404.html", layout: false, status: :not_found
   end
-
```

Comment:
Why did you delete this part?

Reply:
Because there's another function of the same name in application helpers, which I myself refactored in a previous PR. I've now moved that function here, to fix the test errors.

Reply:
Alright this seems good! I will approve once travis passes

Reply:
It got worse :( I'm reverting back to 1st commit on this brach.

Reply:
It failed because you removed the `logged_in_as` method for the application helper. 
Maybe, on this pull request, just edit the current_user_session method and don't change the existing 'log_in_as' methods

Reply:
Oh yeah alright you did it :+1:

Reply:
Yes, that's the reason. But now, we still have 2 `logged_in_as` functions. Maybe that will have to get its own PR. At least, now the PR title is more accurate ;)